src/oatpp/network/monitor/ConnectionInactivityChecker.hpp

```diff
@@ -25,6 +25,7 @@
 #ifndef oatpp_network_monitor_ConnectionInactivityChecker_hpp
 #define oatpp_network_monitor_ConnectionInactivityChecker_hpp
 
+#include "../../core/base/Compiler.hpp"
```

Comment:
`..` isn't allowed in the include paths. 
Please change to "oatpp/core/base/Compiler.hpp"